pkg/identity/numericidentity.go

```diff
@@ -241,6 +244,20 @@ func InitWellKnownIdentities() {
 		fmt.Sprintf("k8s:%s=cilium-operator", api.PolicyLabelServiceAccount),
 		fmt.Sprintf("k8s:%s=%s", api.PolicyLabelCluster, option.Config.ClusterName),
 	})
+
+	// cilium-etcd-operator labels
+	//   k8s:io.cilium.k8s.policy.cluster=default
+	//   k8s:io.cilium.k8s.policy.serviceaccount=cilium-etcd-operator
+	//   k8s:io.cilium/app=etcd-operator
+	//   k8s:io.kubernetes.pod.namespace=<NAMESPACE>
+	//   k8s:name=cilium-etcd-operator
+	WellKnown.add(ReservedCiliumEtcdOperator, []string{
+		"k8s:name=cilium-etcd-operator",
+		"k8s:io.cilium/app=etcd-operator",
+		fmt.Sprintf("k8s:%s=%s", api.PodNamespaceLabel, namespace),
+		fmt.Sprintf("k8s:%s=cilium-etcd-operator", api.PolicyLabelServiceAccount),
+		fmt.Sprintf("k8s:%s=%s", api.PolicyLabelCluster, option.Config.ClusterName),
```

Comment:
We should move the creation of the cluster label into a utility function since it's duplicated across all well-known identities to reduce the amount of boilerplate here. Similarly we can add utility functions for service account, namespace labels, etc.